core/src/main/java/org/apache/carbondata/core/dictionary/client/DictionaryClient.java

```diff
@@ -44,7 +40,7 @@
 
   private DictionaryClientHandler dictionaryClientHandler = new DictionaryClientHandler();
 
-  private ClientBootstrap clientBootstrap;
+  private NioEventLoopGroup workerGroup;
```

Comment:
I need to have `NioEventLoopGroup` to shutdown the server, cannot shutdown from bootstrap